### ISSUE TITLE
Discord: clicking images no longer opens them full size, but instead opens them in a web browser

### ISSUE THREAD

**Issue by briankendall** [getferdi/ferdi#1938]
**Bug description:**
Clicking posted images in Discord no longer shows the image full screen with the background darkened and a "show original" link, as it does in both the desktop app and web app. Instead it opens the image immediately in a web browser. This behavior is the same effect as clicking the "show original" link.

**Steps to reproduce:**

1. Open any channel on a Discord server where someone has posted an image
2. Click the image

**Expected behavior**
The rest of the page darkens and the image opens up at a larger size, with a X button to dismiss it and a link that says "open original".

**Screenshots**
Here is a screen shot of what's supposed to happen when clicking an image, taken from the Discord web app running in Chrome:
<img width="892" alt="Screen Shot 2021-09-14 at 10 46 38 AM" src="https://user-images.githubusercontent.com/7917884/133280318-f4dbbfb7-956a-41e2-ba28-4ecc087b1f06.png">

**Environment:**

- Operating System: macOS 10.14.6
- Ferdi Version: 5.6.2
- Server: Ferdi, using without an account

- Debug information:

```{"host":{"platform":"darwin","release":"18.7.0","screens":[{"id":69734409,"bounds":{"x":0,"y":0,"width":1440,"height":900},"workArea":{"x":0,"y":23,"width":1440,"height":877},"accelerometerSupport":"unknown","monochrome":false,"colorDepth":24,"colorSpace":"{primaries_d50_referred: [[0.6818, 0.3196],  [0.2846, 0.6747],  [0.1549, 0.0554]], transfer:0.0777*x + 0.0000 if x < 0.0450 else (0.9495*x + 0.0495)**2.3955 + 0.0003, matrix:RGB, range:FULL}","depthPerComponent":8,"size":{"width":1440,"height":900},"displayFrequency":60,"workAreaSize":{"width":1440,"height":877},"scaleFactor":2,"rotation":0,"internal":true,"touchSupport":"unknown"}]},"ferdi":{"version":"5.6.2","electron":"13.3.0","installedRecipes":[{"id":"discord","version":"1.4.0"},{"id":"skype","version":"3.2.0"},{"id":"slack","version":"1.3.0"},{}],"devRecipes":[],"services":[{"id":"a2c79c9c-0fc8-4693-a0e2-4a9dd759b5a3","recipe":"skype","isAttached":true,"isActive":false,"isEnabled":true,"isHibernating":false,"hasCrashed":false,"isDarkModeEnabled":false},{"id":"27dcb796-4475-426d-a414-5d0d89a3391b","recipe":"slack","isAttached":true,"isActive":false,"isEnabled":true,"isHibernating":false,"hasCrashed":false,"isDarkModeEnabled":false},{"id":"b4fd6689-0f2c-425e-8c4d-5958bc66cb93","recipe":"slack","isAttached":true,"isActive":false,"isEnabled":true,"isHibernating":false,"hasCrashed":false,"isDarkModeEnabled":false},{"id":"9daecdde-9d8f-45e0-a37f-992100171f15","recipe":"slack","isAttached":true,"isActive":false,"isEnabled":true,"isHibernating":false,"hasCrashed":false,"isDarkModeEnabled":false},{"id":"4c73a83d-a0dc-40c3-b02f-fed839109742","recipe":"slack","isAttached":true,"isActive":false,"isEnabled":true,"isHibernating":false,"hasCrashed":false,"isDarkModeEnabled":false},{"id":"66a5fb6e-9e0b-4572-a153-f501feaf52df","recipe":"slack","isAttached":true,"isActive":false,"isEnabled":true,"isHibernating":false,"hasCrashed":false,"isDarkModeEnabled":false},{"id":"6dd085f1-937f-4974-9d5c-f29172231621","recipe":"discord","isAttached":true,"isActive":true,"isEnabled":true,"isHibernating":false,"hasCrashed":false,"isDarkModeEnabled":false}],"messages":[{"type":"error","info":["error","Warning: Failed prop type: Invalid prop `actions.service` of type `Object` supplied to `_class2`, expected instance of `_class`.\n    in _class2 (created by inject-_class2-with-stores-actions)\n    in inject-_class2-with-stores-actions (created by RouterContext)\n    in RouterContext (created by Router)\n    in Router (created by _class)\n    in _class (created by inject-_class-with-stores-actions)\n    in inject-_class-with-stores-actions\n    in IntlProvider (created by _class)\n    in _class (created by inject-_class-with-stores)\n    in inject-_class-with-stores\n    in Provider"]},{"type":"error","info":["error","Warning: Failed prop type: Invalid prop `actions.settings` of type `Object` supplied to `_class2`, expected instance of `_class`.\n    in _class2 (created by inject-_class2-with-stores-actions)\n    in inject-_class2-with-stores-actions (created by _class2)\n    in div (created by _class2)\n    in div (created by _class2)\n    in _class2 (created by Jss(_class2))\n    in Jss(_class2) (created by _class2)\n    in _class2 (created by Jss(_class2))\n    in Jss(_class2) (created by _class2)\n    in ThemeProvider (created by _class2)\n    in _class2 (created by inject-_class2-with-stores-actions)\n    in inject-_class2-with-stores-actions (created by RouterContext)\n    in RouterContext (created by Router)\n    in Router (created by _class)\n    in _class (created by inject-_class-with-stores-actions)\n    in inject-_class-with-stores-actions\n    in IntlProvider (created by _class)\n    in _class (created by inject-_class-with-stores)\n    in inject-_class-with-stores\n    in Provider"]},{"type":"error","request":{"result":null,"wasExecuted":true,"method":"getInfo"},"error":{},"response":{},"server":"You are using Ferdi without a server"},{"type":"error","request":{"result":null,"wasExecuted":true,"method":"features"},"error":{},"response":{},"server":"You are using Ferdi without a server"},{"type":"error","request":{"result":null,"wasExecuted":true,"method":"all"},"error":{},"response":{},"server":"You are using Ferdi without a server"}],"workspaces":[],"windowSettings":{"width":977,"height":877,"x":0,"y":23,"displayBounds":{"x":0,"y":0,"width":1440,"height":900},"isMaximized":false,"isFullScreen":false},"settings":{"autoLaunchInBackground":false,"runInBackground":false,"reloadAfterResume":true,"enableSystemTray":false,"startMinimized":false,"confirmOnQuit":false,"minimizeToSystemTray":false,"closeToSystemTray":false,"privateNotifications":false,"clipboardNotifications":true,"notifyTaskBarOnMessage":false,"showDisabledServices":true,"showMessageBadgeWhenMuted":true,"showDragArea":false,"enableSpellchecking":true,"spellcheckerLanguage":"en-us","darkMode":true,"locale":"en-US","fallbackLocale":"en-US","beta":false,"isAppMuted":false,"enableGPUAcceleration":true,"server":"You are using Ferdi without a server","predefinedTodoServer":"https://app.franztodos.com","autohideMenuBar":false,"lockingFeatureEnabled":false,"locked":false,"lockedPassword":"******","useTouchIdToUnlock":true,"scheduledDNDEnabled":false,"scheduledDNDStart":"17:00","scheduledDNDEnd":"09:00","hibernateOnStartup":true,"hibernationStrategy":300,"wakeUpStrategy":300,"inactivityLock":0,"automaticUpdates":false,"showServiceNavigationBar":false,"universalDarkMode":false,"userAgentPref":"","adaptableDarkMode":false,"accentColor":"#7367f0","serviceRibbonWidth":68,"iconSize":20,"sentry":false,"nightly":false,"navigationBarBehaviour":"never","searchEngine":"duckDuckGo","useVerticalStyle":false,"alwaysShowWorkspaces":false,"liftSingleInstanceLock":false,"serviceLimit":5,"hibernate":false,"todoServer":"isUsingCustomTodoService","noUpdates":true,"customTodoServer":""},"features":{"isServiceProxyEnabled":true,"isWorkspaceEnabled":true,"isTodosEnabled":true,"isSettingsWSEnabled":false},"user":"82c1cf9d-ab58-4da2-b55e-aaa41d2142d8"}}```

**Comment by briankendall** [comment]
I have some additional information: I just updates another mac system to 5.6.2. After the update, clicking an image in Discord behaved correctly. Then a banner appeared at the bottom saying that my services had been updated, and I clicked the reload button. Now after that, clicking images erroneously opens them in a browser.

**Comment by gschaffner** [comment]
I'm also seeing this behavior on Manjaro Linux with Ferdi 5.6.2. via `ferdi-bin` (AUR), so I don't think it's a Mac-exclusive bug.

**Comment by briankendall** [comment]
The problem is in the Discord recipe. I've confirmed that it was introduced with recipe version 1.2.0 by [fe6130](https://github.com/getferdi/recipes/commit/fe613088a813394760ca27afeee554cfa7290d67).

Should I recreate this issue in the recipe repo?

**Comment by Gaboris** [comment]
I just updated Ferdi and Discord as well and saw this bug so I wasn't sure what caused it. Thanks for the quick PR I hope it'll get applied soon. 👍

**Comment by vraravam** [comment]
This will be in the upcoming nightly (`v5.6.3-nightly.4`) in an hour or so. Please upgrade and verify.

**Comment by m-vlad** [comment]
Considering this is an issue with the stable version I reckon it should be fixed in the stable channel as well

**Comment by vraravam** [comment]
@m-vlad - though that is true, it doesn't mean that we will release to stable for every bug that gets fixed. Please be assured that the nightlies are also quite stable (and you can turn off auto-updates to stop the reminders of new updates).

**Comment by m-vlad** [comment]
The issue is it's been a month now and the "stable" version still has this issue. Stable versions shouldn't have these kinds of bugs. I am also unable to switch to nightly because I installed ferdi through choco and choco doesn't have the nightly version available.

**Comment by vraravam** [comment]
@m-vlad - in a commercial product, a stable version can be quickly released. For any OSS software, its upto the prerogative of the maintainers to release the software. (Ironically, that's the reason I started contributing to Ferdi in the first place!)

That being said, we released a beta version just a week ago, and if there's no significant movement in the coming week, I would be willing to release a new version to fix such niggles.

btw - the nightlies are quite stable, and we rarely have to call out an urgent fix on them.
